docs/source/guide/start.md

```diff
@@ -10,22 +10,125 @@ After you install Label Studio, start the server to start using it.
+| `--password` | `LABEL_STUDIO_PASSWORD` | Password to use for the default user. |
+| `--username` | `LABEL_STUDIO_USERNAME` | Username to use for the default user. |
+| `--agree-fix-sqlite` | N/A | Automatically agree to let Label Studio fix SQLite issues when using Python 3.6–3.8 on Windows operating systems. | 
+
+
+## Run Label Studio on localhost with default settings
+By default, Label Studio uses the `0.0.0.0` address to start. If you want to use `localhost` instead, use the following command:
+```bash
+label-studio start --internal-host localhost
+```
 
-The SQLite database works well for projects with tens of thousands of labeling tasks. If you want to annotate millions of tasks or anticipate a lot of concurrent users, use a PostgreSQL database. See [Install and upgrade Label Studio](install.html#PostgreSQL-database) for more.  
+Or, set the following environment variable:
+```
+LABEL_STUDIO_INTERNAL_HOST = localhost
+```
 
-For example, if you import data while labeling is being performed, labeling tasks can take more than 10 seconds to load and annotations can take more than 10 seconds to perform. If you want to label more than 100,000 tasks with 5 or more concurrent users, consider using PostgreSQL or another database with Label Studio. 
+## Run Label Studio on localhost with a different port
+By default, Label Studio runs on port 8080. If that port is already in use or if you want to specify a different port, start Label Studio with the following command:
+```bash
+label-studio start --port <port>
+```
 
-## Command line arguments for starting Label Studio
+For example, start Label Studio on port 9001:
+```bash
+label-studio start --port 9001
+```
 
-You can specify project config, machine learning backend and other options using the command line interface. Run `label-studio start --help` to see all available options. Many command line arguments are deprecated and removed in version 1.0.0. 
+Or, set the following environment variable:
+```
+LABEL_STUDIO_PORT = 9001
+```
+
+## Run Label Studio on Docker with a different port
+
+To run Label Studio on Docker with a port other than the default of 8080, use the port argument when starting Label Studio on Docker. For example, to start Label Studio in a Docker container accessible with port 9001, run the following: 
+```bash
+docker run -it -p 9001:8080 -v `pwd`/mydata:/root/.local/share/label-studio/ heartexlabs/label-studio:latest label-studio
+```
 
-* The Label Studio web server always uses the `0.0.0.0` address to start. If you need to change it to `localhost`, set `--internal-host` from console arguments to `localhost` and the web server starts at `localhost`.
+Or, if you're using Docker Compose, update the `docker-compose.yml` file that you're using to expose a different port for the app. For example, this portion of the `docker-compose.yml` file exposes port 9001 instead of port 8080 for running Label Studio:
```

Comment:
For the docker-compose, [this line](https://github.com/heartexlabs/label-studio/blob/release/1.0.1/docker-compose.yml#L8) should be fixed as `9001:80`. Also we should mention that in our docker-compose.yml nginx server is used as a front and proxies requests to the app server